files/python/requirements.txt

```diff
@@ -1,7 +1,8 @@
 # The python packages required to make the image work
+RPi.GPIO>=0.7.0,<1
 git+https://github.com/srobo/sr-robot.git@sr2024
 kchd
-astoria
+git+https://github.com/srobo/astoria.git@wbb/sr2024-fixes
```

Comment:
We should probably get this merged first.

Reply:
Because I was originally testing with the v0.11.1 release I made a branch from that release for the changes needed. I haven't tested with all the bits that have been added to master since the release, so I'm not sure about how we want to handle releasing that.